File: ios/NotificationScheduler.m
//
//  NotificationScheduler.m
//  ScheduledNotifications
//
//  Created by Muhammad Ali on 10/02/2020.
//  Copyright © 2020 Facebook. All rights reserved.
//

#import "React/RCTBridgeModule.h"

@interface RCT_EXTERN_MODULE(NotificationScheduler, NSObject)

RCT_EXTERN_METHOD(scheduleNotification:(NSString *)uuid message:(NSString *)message datetime:(nonnull NSNumber *)datetime)

RCT_EXTERN_METHOD(removeNotification:(NSString *)uuid)

@end

File: ios/NotificationScheduler.swift
//
//  NotificationScheduler.swift
//  ScheduledNotifications
//
//  Created by Muhammad Ali on 10/02/2020.
//  Copyright © 2020 Facebook. All rights reserved.
//

import Foundation

@objc(NotificationScheduler)
class NotificationSchedular: NSObject {
  
  @objc
  func scheduleNotification(_ uuid: String, message: String, datetime: NSNumber) -> Void {
    print("@RNSN -> Schedule Notification: ", message, uuid, datetime)
  }
  
  @objc
  func removeNotification(_ uuid: String) -> Void {
    print("@RNSN -> Remove notification with ID: ", uuid)
  }
}
